README.md

```diff
@@ -4,25 +4,29 @@ Copyright 2016-2020 DMTF. All rights reserved.
 
 ## About
 
-The Redfish Mockup Server serves Redfish requests against a Redfish mockup.
-The server runs at a specified IP address and port or at `127.0.0.1:8000`, which is the default IP address and port.
+The Redfish Mockup Server serves Redfish requests against a Redfish mockup. The server runs at either a specified IP address and port or the default IP address and port, `127.0.0.1:8000`.
 
-Sample mockups published by the DMTF can be found here: [https://www.dmtf.org/dsp/DSP2043](https://www.dmtf.org/dsp/DSP2043).
-The [Redfish-Mockup-Creator](https://github.com/DMTF/Redfish-Mockup-Creator) can be used to create a mockup from an existing service.
+You can find DMTF-published sample mockups at [All Published Versions of DSP2043](https://www.dmtf.org/dsp/DSP2043 "https://www.dmtf.org/dsp/DSP2043").
+
+To create a mockup from a service, use the [Redfish-Mockup-Creator](https://github.com/DMTF/Redfish-Mockup-Creator "https://github.com/DMTF/Redfish-Mockup-Creator").
 
 ## Requirements
 
-If running the mockup server natively on your system:
-* Install [Python 3](https://www.python.org/downloads/) and [pip](https://pip.pypa.io/en/stable/installing/)
+To run the mockup server natively on your system:
+
+* Install [Python 3](https://www.python.org/downloads/ "https://www.python.org/downloads/") and [pip](https://pip.pypa.io/en/stable/installing/ "https://pip.pypa.io/en/stable/installing").
 * Install required Python packages: `pip install -r requirements.txt`
+* See the [Native system example](#native-system-example).
```

Comment:
The stuff that you added following "Usage" was confusing to me - but I think this was the intent - a native system example, and then the Docker build step and a docker container example.